doc/cascadia/profiles.schema.json

```diff
@@ -616,7 +539,7 @@
         },
         "padding": {
           "default": "8, 8, 8, 8",
-          "description": "Sets the padding around the text within the window. Can have three different formats: \"#\" sets the same padding for all sides, \"#, #\" sets the same padding for left-right and top-bottom, and \"#, #, #, #\" sets the padding individually for left, top, right, and bottom.",
+          "description": "Sets the padding around the text within the window. Can have three different formats:\n -\"#\" sets the same padding for all sides \n -\"#, #\" sets the same padding for left-right and top-bottom\n -\"#, #, #, #\" sets the padding individually for left, top, right, and bottom.",
```

Comment:
Proud of this one too...
![image](https://user-images.githubusercontent.com/11050425/79016743-978f2000-7b24-11ea-809f-cf1f1d2e67a0.png)

Reply:
looks sharp